nachomemes/bot.py

```diff
@@ -265,57 +327,31 @@ async def save(ctx: Context):
             raise RuntimeError("computer says no")
         value = ctx.message.content
         value = ctx.message.content[value.index("save")+4:].strip().strip('`')
-        message = STORE.save_meme(config.guild_id, json.loads(value))
-        await ctx.send(textwrap.dedent(f"```{message}```"))
-    except JSONDecodeError:
-        await ctx.send(textwrap.dedent(f"```invalid JSON provided```"))
+        await ctx.send(embed=Embed(
+            title = "Saving meme",
+            description = STORE.save_meme(config.guild_id, json.loads(value))
+        ))
+    except JSONDecodeError as ex:
+        await report(ctx, ex, "Invalid JSON provided")
     except Exception as ex:
         await report(ctx, ex)
 
+
 @memebot.command(description="dump template json")
 async def dump(ctx: Context, template_name: str=None):
-    if not template_name:
-        return await ctx.send(textwrap.dedent(f"```No template name provided```"))
     try:
         config: GuildConfig = STORE.guild_config(ctx.guild)
         if not config.can_edit(_get_member(ctx)):
             raise RuntimeError("computer says no")
-        template = STORE.best_match(template_name, config.guild_id)
-        message = json.dumps(template, cls=TemplateEncoder)
-        await ctx.send(textwrap.dedent(f"```{message}```"))
-    except TemplateError as ex:
-        await ctx.send(textwrap.dedent(f'```No template matching "{template_name}" found```'))
+        template = STORE.best_match(config.guild_id, template_name, )
+        await ctx.send(embed=Embed(
+            title = "Exporting meme",
+            description = json.dumps(template, cls=TemplateEncoder)
```

Comment:
Does the description need to be in a codeblock?

Reply:
sure